html/components/data-management-tab.js

```diff
@@ -0,0 +1,249 @@
+			buttons: {
+				"Yes, Remove": function() {
+					div.find("p.Warning,p.Confirm").text("");
+					div.find("p.Status").text("Removing file...");
+					$.post("/Media/remove", {
+						media: format,
+						type: type,
+						filename: file
+					}).fail(function(jqxhr, textStatus, errorThrown) {
+						div.find("p.Status").text("Error removing media file: "+jqxhr.responseText);
+						div.dialog("option", "buttons", { Close: function() { div.dialog("close"); } });
+					}).done(function(data, textStatus, jqXHR) {
+						div.dialog("close");
+					});
+				},
+				"No": function() {
+					div.dialog("close");
+				}
+			}
+		});
+	}
+	
+	var uploadDialogTemplate = $('<div>').addClass('UploadMediaDialog')
+		.append($('<p>').append($('<div>').addClass('Upload')))
+		.append($('<p>').append($('<a>').addClass('File').text('File: '))
+				.append($('<input type="file" multiple>').addClass('File')))
+		.append($('<p>').append($('<a>').addClass('Status')));
+
+	function createUploadMediaDialog(format, type) {
+		var div = uploadDialogTemplate.clone(true);
+		var uploader = div.find("div.Upload").fileupload({
+			url: "/Media/upload",
+			dropZone: null,
+			singleFileUploads: false
+		});
+		var inputFile = div.find("input:file.File");
+		var uploadFunction = function() {
+			var data = { files: $(this).find("input:file.File")[0].files };
+			var length = data.files.length;
+			var statustxt = "file"+(length>1?"s":"");
+			uploader.fileupload("option", "formData", [
+				{ name: "media", value: format },
+				{ name: "type", value: type }
+				]);
+			uploader.fileupload("send", data)
+			.done(function(data, textStatus, jqxhr) {
+				div.find("a.Status").text(data);
+			})
+			.fail(function(jqxhr, textStatus, errorThrown) {
+				div.find("a.Status").text("Error while uploading : "+jqxhr.responseText);
+			})
+			.always(function() {
+				var newInputFile = inputFile.clone(true).insertAfter(inputFile);
+				inputFile.remove();
+				inputFile = newInputFile.change();
+			});
+			uploader.fileupload("option", "formData", []);
+		};
+		var closeFunction = function() { $(this).dialog("close"); };
+		var buttonsCloseOnly = { Close: closeFunction };
+		var buttonsUploadClose = { Upload: uploadFunction, Close: closeFunction };
+
+		div.dialog({
+			title: "Upload media "+format+" : "+type,
+			modal: true,
+			width: 700,
+			close: function() { $(this).dialog("destroy").remove(); },
+			buttons: buttonsCloseOnly
+		});
+		inputFile.change(function() {
+			var files = this.files;
+			if (!files || !files.length) {
+				div.dialog("option", "buttons", buttonsCloseOnly);
+				return;
+			}
+			div.dialog("option", "buttons", buttonsUploadClose);
+		}).change();
+	}
+	
+	function createGameDataTab(tab) {
+		var iframe = $('<iframe src="/game-data">').attr('id', 'GameData');
+		tab.append($('<table>')
+				.append($('<td>').addClass('Info').text('To download, right-click and Save - to view JSON, left-click'))
```

Comment:
Let's just have a link to the actual directory, and add a README if docs are needed. No need for a reload button if you can use the browser's.